popgen/stationarity.go

```diff
@@ -174,7 +174,8 @@ func AlleleFrequencyProbability(i int, n int, alpha float64, binomMap [][]float6
 //AfsLikelihood returns P(Data|alpha), or the likelihood of observing a particular allele frequency spectrum given alpha, a vector of selection parameters.
 func AFSLikelihood(afs AFS, alpha []float64, binomMap [][]float64) float64 {
 	var answer float64 = 0.0
-	for j := 1; j < len(afs.sites); j++ {
```

Comment:
Not sure what caused this error, it doesn't make a lot of sense. Great correction